app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MetricsController < ApplicationController
+  def index
+    @hash = { "labels" => [],
+      "datasets" =>[
+      fillColor: "rgb(1,97,158)",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.